wallet/src/de/schildbach/wallet/ui/EditProfileActivity.kt

```diff
@@ -66,9 +63,13 @@ class EditProfileActivity : BaseMenuActivity() {
 
     private lateinit var editProfileViewModel: EditProfileViewModel
     private lateinit var selectProfilePictureSharedViewModel: SelectProfilePictureSharedViewModel
+    private lateinit var externalUrlSharedViewModel: ExternalUrlProfilePictureViewModel
+
     private var isEditing: Boolean = false
     private var defaultAvatar: TextDrawable? = null
 
+    private var prosilePictureChanged = false
```

Comment:
`prosilePictureChanged` - **profile** is mispelled.

Reply:
Fixed